channels/candidate-4.4.yaml

```diff
@@ -1,5 +1,6 @@
 name: candidate-4.4
 versions:
+- 4.3.14+amd64
```

Comment:
oops ,not sorted

Reply:
I had assumed this was a requirement of our new robot masters.